packages/cli/src/commands/setup/auth/templates/auth.js.template

```diff
@@ -36,19 +36,24 @@ export const getCurrentUser = async (
  *
  * @returns {boolean} - If the currentUser is authenticated
  */
-export const isAuthenticated = () => {
+export const isAuthenticated = (): boolean => {
   return !!context.currentUser
 }
 
+/**
+ * When checking role membership, roles can be a single value, a list, or none
```

Comment:
```suggestion
 * When checking role membership, roles can be a single value, a list, or none. You can use Prisma enums too, just import your enum type from `@prisma/client`
```

Reply:
Made a few JSDoc updates in line with this suggestion.